packages/teams-js/src/public/stageView.ts

```diff
@@ -115,4 +115,39 @@ export namespace stageView {
   export function isSupported(): boolean {
     return ensureInitialized(runtime) && runtime.supports.stageView ? true : false;
   }
+
+  export namespace stageClose {
+    /**
+     * Closes the current stage view.
+     * @returns Promise that resolves or rejects with an error once the stage view is closed.
+     */
+    export function closeCurrentWindow(): Promise<void> {
```

Comment:
This function should have a complete suite of unit tests added to stageView.spec.ts